controller/types.go

```diff
@@ -25,36 +25,36 @@ var registeredResultMetrics []string
 // Declare the fixed chaos metrics. Dynamic (testStatus) metrics are defined in metrics()
 var (
 	EngineTotalExperiments = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: "chaosEngine",
-		Subsystem: "engine",
-		Name:      "engine_total_experiments",
+		Namespace: "chaosengine",
+		Subsystem: "",
+		Name:      "experiment_count",
```

Comment:
from my point, it's better to name it: `experiments_count`, because we've counted all of them deployed in a cluster.

Reply:
Thanks for the review @Jasstkn , will take care of this in next PR.